plugins/modules/dcnm_vrf.py

```diff
@@ -369,24 +456,73 @@ def update_attach_params(self, attach, vrf_name, deploy, vlanId=''):
         for ip, ser in self.ip_sn.items():
             if ip == attach['ip_address']:
                 serial = ser
+                self.serial = ser
 
         if not serial:
             self.module.fail_json(msg='Fabric: {} does not have the switch: {}'
                                   .format(self.fabric, attach['ip_address']))
 
         role = self.inventory_data[attach['ip_address']].get('switchRole')
+        self.role = role
+
         if role.lower() == 'spine' or role.lower() == 'super spine':
             msg = 'VRFs cannot be attached to switch {} with role {}'.format(attach['ip_address'], role)
             self.module.fail_json(msg=msg)
 
+        ext_values = {}
+        if attach['vrf_lite']:
+            '''Before apply the vrf_lite config, need double check if the swtich role is started wth Border'''
+            if (role.lower() != 'border' and
```

Comment:
Can you simplify by using a regular expression here and just check for `boarder` in the role name?

```python
>>> if not re.search(r'border', role.lower()):
...     print('VRF LITE cannot be attached')
... 
>>> role = 'spine'
>>> 
>>> if not re.search(r'border', role.lower()):
...     print('VRF LITE cannot be attached')
... 
VRF LITE cannot be attached
>>> 
```

Reply:
done.